web-wallet/src/lib/dusk/number/createNumberFormatter.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Creates a locale aware number formatter
+ *
+ * @param {String} locale A BCP 47 language tag
+ * @param {Number|Undefined} digits The minimum fraction digits that should display
```

Comment:
You say "minimum" her, but the the digits are set to `maximumFractionDigits` in the options.